tests/unit/FilingHistoryList.spec.ts

```diff
@@ -586,7 +586,8 @@ describe('Filing History List - incorporation applications', () => {
     expect(wrapper.find('.filing-label').text()).toContain('Incorporation Application')
     expect(wrapper.find('.filing-label').text()).toContain('Numbered Benefit Company')
     const spans = wrapper.findAll('.list-item__subtitle span')
-    expect(spans.at(0).text()).toBe('FILED AND PENDING (filed by Full Name on 2020-05-06)')
+    expect(spans.at(0).text())
+      .toBe('FILED AND PENDING (filed by Full Name on 2020-05-06) | EFFECTIVE as of 2020-05-06')
```

Comment:
Are these tests in here for all the other variations (eg, without EFFECTIVE phrase)?

Reply:
There is yes :) Not all tests were impacted, ie the instances where there was no effective date it is confirmed to be NOT appended.